app/javascript/gobierto_visualizations/modules/costs_controller.js

```diff
@@ -150,23 +150,28 @@ export class CostsController {
       "costdirfin"
     ];
 
-    const population = ["126988"];
+    const population = ["128291", "129661"];
+
+    let yearsCosts = [...new Set(rawData.map(item => item.any_))];
 
     for (let index = 0; index < rawData.length; index++) {
       let d = rawData[index];
 
-      if (d["any_"] === "2019") {
-        d["population"] = population[0];
-      }
-
+      d["population"] = getPopulation(d["any_"]);
       for (let amounts = 0; amounts < amountStrings.length; amounts++) {
         d[amountStrings[amounts]] = convertStringToNumbers(
           d[amountStrings[amounts]]
         );
       }
     }
 
-    let yearsCosts = [...new Set(rawData.map(item => item.any_))];
+    function getPopulation(year) {
```

Comment:
This function is totally worthless:
1. There is only one use (L160)
2. It's running over and over through the yearsCosts array
3. yearsCosts array is defined outside the function

Why not this? The current code is quite hard to understand
```
const ix = yearsCosts.findIndex(x => x === d["any_"])
d.population = population[ix]
```

Reply:
Much better, done.